README.md

```diff
@@ -17,6 +17,10 @@ There are currently a few projects using these resources or related to this proj
 If you'd like to contribute, please fork the repository and make changes as
 you'd like. You might want to add bees or flowers for your region following our schemas. Pull requests are warmly welcome.
 
+## License
+
+The code in this repository is released under the MIT license. The illustrations themselves are released by Olivia Carlson under +Creative Commons Attribution-NonCommercial 4.0 International (CC BY-NC 4.0). Other photographs or images are released under their own listed attributions.
```

Comment:
```suggestion
The contents of this repository is released under the MIT license except as otherwise noted. The illustrations in the `/art` directory is released by Olivia Carlson under [Creative Commons Attribution-NonCommercial 4.0 International (CC BY-NC 4.0)](https://creativecommons.org/licenses/by-nc/4.0/). The images in the `/photos` directory are subject to the copyright and licensing described in the associated JSON files.
```